src/commands/mod.rs

```diff
@@ -289,6 +291,48 @@ impl DaemonControl {
         GetAddressResult::new(address)
     }
 
+    /// list addresses
+    pub fn list_addresses(&self, start_index: u32, count: u32) -> GetAddressesResult {
```

Comment:
To make those parameters optional it's better to have them as `Option`s rather than using a magic "0" value.

Reply:
do you also want `count` to be optionnal in rpc call?

Reply:
I think both should be. `start_height` should default to `0` and `count` to the max of the change and receive indexes.